test/scope/ScopeModel.spec.js

```diff
@@ -29,6 +31,35 @@ ava('creates RadarTargetCollection on instantiation', (t) => {
     t.true(model.radarTargetCollection instanceof RadarTargetCollection);
 });
 
+ava('#ptlLength returns #_ptlLength', (t) => {
+    const model = new ScopeModel();
+    const expectedResult = 17;
+    model._ptlLength = expectedResult;
+    const result = model.ptlLength;
+
+    t.true(result === expectedResult);
```

Comment:
if you want to use `t.is()` for these instead, be my guest.  `.is()` will provide better error logging and also will check identity.  Not required, just calling it out 😉

Reply:
I like consistency, so I'll add an issue to find-and-replace the whole repo, and see if anything bad happens rather than just doing a few here.

Reply:
Created #1393.